terraform/modules/siri-consumer-api/main.tf

```diff
@@ -44,6 +47,98 @@ resource "aws_api_gateway_rest_api" "siri_consumer_api" {
   }
 }
 
+resource "aws_cloudwatch_log_group" "siri_consumer_api_log_group" {
+  name              = "${var.api_name}-${var.environment}"
+  retention_in_days = var.environment == "prod" ? 90 : 30
+}
+
+resource "aws_api_gateway_stage" "siri_consumer_api_stage" {
+  rest_api_id           = aws_api_gateway_rest_api.siri_consumer_api.id
+  deployment_id         = aws_api_gateway_deployment.siri_consumer_api_deployment.id
+  stage_name            = "v1"
+  cache_cluster_enabled = true
+  cache_cluster_size    = var.environment == "prod" ? 1.6 : 0.5
+
+  dynamic "access_log_settings" {
+    for_each = var.environment != "local" ? ["apply"] : []
+
+    content {
+      destination_arn = aws_cloudwatch_log_group.siri_consumer_api_log_group.arn
+      format = jsonencode({
+        requestId         = "$context.requestId"
+        extendedRequestId = "$context.extendedRequestId"
+        ip                = "$context.identity.sourceIp"
+        caller            = "$context.identity.caller"
+        user              = "$context.identity.user"
+        requestTime       = "$context.requestTime"
+        requestTimeEpoch  = "$context.requestTimeEpoch"
+        resourcePath      = "$context.resourcePath"
+        httpMethod        = "$context.httpMethod"
+        status            = "$context.status"
+        protocol          = "$context.protocol"
+        responseLength    = "$context.responseLength"
+      })
+    }
+  }
+}
+
+resource "aws_api_gateway_rest_api_policy" "siri_consumer_api_resource_policy" {
+  count = var.private ? 1 : 0
+
+  rest_api_id = aws_api_gateway_rest_api.siri_consumer_api.id
+  policy = jsonencode({
+    Version = "2008-10-17"
+    Statement = [
+      {
+        "Effect" : "Deny",
+        "Principal" : "*",
+        "Action" : "execute-api:Invoke",
+        "Resource" : [
+          "arn:aws:execute-api:${var.aws_region}:${var.account_id}:${aws_api_gateway_rest_api.siri_consumer_api.id}/*/${aws_api_gateway_method.siri_vm_api_downloader_method.http_method}${aws_api_gateway_resource.siri_vm_api_downloader_resource.path}",
+          "arn:aws:execute-api:${var.aws_region}:${var.account_id}:${aws_api_gateway_rest_api.siri_consumer_api.id}/*/${aws_api_gateway_method.siri_vm_api_stats_method.http_method}${aws_api_gateway_resource.siri_vm_api_stats_resource.path}"
```

Comment:
I think the siri-sx downloader resource needs adding to the policy here and on line 116